extensions/amp-analytics/0.1/amp-analytics.js

```diff
@@ -56,6 +56,16 @@ const WHITELIST_EVENT_IN_SANDBOX = [
   AnalyticsEventType.HIDDEN,
 ];
 
+/**
+ * @param {!Window} win
+ * @return {number} A timestamp relative to navigationStart or 0 if the
+ *     Navigation Timing API is not supported. The timestamp is in milliseconds
+ *     with sub-millisecond precision (so the value is not necessarily an
+ *     integer).
+ */
+function now(win) {
```

Comment:
`now` is a confusing name with `performance.now` vs `Date.now()`. Can we name it something else? 
Are we trying to polyfill `performance.now` with best effort? by doing `Date.now() - INIT_TIME`

Reply:
Went with `performanceTimestamp`. I don't need to polyfill it since it's only used alongside resource timing ATM.

Reply:
Not polyfill. But try best effort to return a reasonable performance.now() value.
I think you want to use `performance.now` because you want to update `resourceTimingSpec['responseAfter']` to the last report time, so that an entry won't be reported twice. 
However if `performance.now()` is not available,` resourceTimingSpec['responseAfter']` will always equals to the user specified `resourceTimingSpec['responseAfter']`. And an entry will be reported multiple times.